src/main/java/com/treasuredata/client/TDHttpClient.java

```diff
@@ -205,15 +188,7 @@ ObjectMapper getObjectMapper()
 
     public void close()
     {
-        synchronized (this) {
-            try {
-                httpClient.stop();
-            }
-            catch (Exception e) {
-                logger.error("Failed to terminate Jetty client", e);
-                throw Throwables.propagate(e);
-            }
-        }
+        // No need to close OkHttp client
```

Comment:
`com.facebook.presto.cli.QueryRunner` has some cleanup code for OkHttpClient. Don't we need it also? 
```java
    @Override
    public void close()
    {
        httpClient.dispatcher().executorService().shutdown();
        httpClient.connectionPool().evictAll();
    }
```

Reply:
I think we should pick this clean-up code. Thanks!